src/javascript/app/pages/cashier/cashier.js

```diff
@@ -252,7 +252,22 @@ const Cashier = (() => {
                     $('#view_payment_methods').attr('href', previous_href.concat('?anchor=cryptocurrency'));
                 } else {
                     $('.normal_currency').setVisibility(1);
+                    $('#change-account-btn').off('click').on('click', (e)=>{
+                        e.stopPropagation();
+                        $('.main-account').trigger('click');
+                        $('html, body').animate({ scrollTop: 0 }, 'slow');
+                    });
                 }
+                
+                if (!Client.isLoggedIn()) return;
```

Comment:
This is within the if statement `if (Client.isLoggedIn()) {` on line 214, hence not required here.

Reply:
removed this string